pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -135,7 +139,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			Rules: []authorizationapi.PolicyRule{
 				{
 					Verbs:     util.NewStringSet("get"),
-					Resources: util.NewStringSet("imagestreams"),
```

Comment:
A comment that this is checked by `verifyImageStreamAccess` might be helpful for future generations